### DISCUSSION TITLE
General question

### DISCUSSION THREAD

**Discussion by Quickspell** [charliie-dev/nvimdots.lua#11] [Q&A]
Firstly I'd like to say you have one hell of and IDE, 

Failed to run `config` for none-ls.nvim

.../.config/nvim/lua/modules/configs/completion/null-ls.lua:42: attempt to index field 'shellcheck' (a nil value)

# stacktrace:
  - .config/nvim/lua/modules/configs/completion/null-ls.lua:42 _in_ **config**
  
  
I keep getting this ^ I was wondering if this is something you fixed a while ago because I believe I installed an earlier version? (not sure)

 is there away for me to make an autocomand that makes a call to the github repository to grab any updates automatically? 
 
I am a bit new to but i am enjoying arch and learning. I am too much of a berserk fan to passup on your config, I have been trying a to make my own but as I said couldn't resist.

I'm sure half of this is a dumb question but there's only one way to learn, I couldn't find anything digging through all the info

**Comment by charliie-dev** [comment]
Hi, quickspell!

I'm glad that u enjoy my nvim config!
ys, i did also noticed this issue on my side, but i'm a bit busy ATM so i can't fix this bug ASAP.
i think i'll push some commit next week if i got time! 
hope u can wait unitil then, sorry for causing inconvenience.

**Comment by charliie-dev** [comment]
Hi, I just fixed the issue and migrate my config to v0.10 nightly at [001142b](https://github.com/CharlesChiuGit/nvimdots.lua/commit/001142b8f5fbdb7d66b2d3f1ac7258806f9d86b4)

let me know if theirs still any thing wrong!
